### PR TITLE
Feature/3140 token detection eth mainnet

### PR DESCRIPTION
**Pull request by Cal-L**
**Description**

#3140 

**Checklist**

* [ ] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

**Issue**

Resolves #???

### CODE REVIEW

File: app/components/UI/AssetSearch/index.tsx [hunk 1]
```diff
@@ -19,31 +19,37 @@ const styles = StyleSheet.create({
...
+		{ name: 'name', weight: 0.5 },
+		{ name: 'symbol', weight: 0.5 },
+	],
 });
 
-type Props = {
+interface Props {
 	onSearch: ({ results, searchQuery }: { results: TokenListToken[]; searchQuery: string }) => void;
-};
+	onFocus: () => void;
```
Comment (sethkfman) [inline app/components/UI/AssetSearch/index.tsx:47]: Could you add a quick description for the reason of adding onFocus & onBlur?

File: app/components/UI/Navbar/index.js [hunk 2]
```diff
@@ -170,15 +170,22 @@ export function getNavigationOptionsTitle(title, navigation) {
...
-		headerLeft: () => (
-			<TouchableOpacity onPress={navigationPop} style={styles.backButton} testID={'title-back-arrow-button'}>
-				<IonicIcon
-					name={Device.isAndroid() ? 'md-arrow-back' : 'ios-arrow-back'}
-					size={Device.isAndroid() ? 24 : 28}
-					style={styles.backIcon}
-				/>
-			</TouchableOpacity>
-		),
+		headerRight: () =>
```
Comment (sethkfman) [inline app/components/UI/Navbar/index.js:173]: Is this just to have a more native iOS UX?

Comment (Cal-L) [reply app/components/UI/Navbar/index.js:173]: Nope, I've chatted with @yanrong-chen about this and it's just to move towards a more consistent close button. We currently have different ways of showing "close" at the moment. In the logic, you'll notice that headerRight only renders if a boolean passed in is true (which is when you access experimental from the blue banner on the search page).

Comment (Cal-L) [reply app/components/UI/Navbar/index.js:173]: cc: @sethkfman

**Review by sethkfman — APPROVED** [review]
LGTM

**Review by cortisiko — APPROVED** [review]
🌮 🌮 🌮 🌮

**Pull request merged**